recipe/meta.yaml

```diff
@@ -36,16 +36,16 @@ requirements:
     - yarn
     - wget
     - ripgrep
-    - coreutils  # [osx]
-    - patchelf  # [linux]
+    - coreutils                      # [osx]
+    - patchelf                       # [linux]
     - unzip
     - nodejs
     - {{ cdt('mesa-libgl-devel') }}  # [linux]
     - {{ cdt('mesa-dri-drivers') }}  # [linux]
-    - {{ cdt('libselinux') }}  # [linux]
-    - {{ cdt('libxdamage') }}  # [linux]
-    - {{ cdt('libxxf86vm') }}  # [linux]
-    - {{ cdt('libxext') }}     # [linux]
+    - {{ cdt('libselinux') }}        # [linux]
+    - {{ cdt('libxdamage') }}        # [linux]
+    - {{ cdt('libxxf86vm') }}        # [linux]
+    - {{ cdt('libxext') }}           # [linux]
```

Comment:
these cdts should be regular conda packages

see https://github.com/conda-forge/qt-main-feedstock/pull/139/files